portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -68,24 +72,33 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
   }
 
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    Query query = new Query(Constants.ENTITY_PARAM).addSort(Constants.TIMESTAMP_PARAM, 
+    Query query = new Query(Constants.COMMENT_ENTITY_PARAM).addSort(Constants.TIMESTAMP_PARAM, 
```

Comment:
A more readable way (and in line with the style guide) to format this line is:

```java
    Query query = new Query(Constants.COMMENT_ENTITY_PARAM)
        .addSort(Constants.TIMESTAMP_PARAM, SortDirection.DESCENDING);
```

When lines need to wrap, then each "segment" (e.g. function call) goes on its own line.